lib/cli/src/generators/SVELTE/template-csf/stories/welcome.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+
+</script>
+
+<style>
+  main {
+    padding: 15px;
+    line-height: 1.4;
+    font-family: 'Helvetica Neue', Helvetica, 'Segoe UI', Arial, freesans, sans-serif;
+    background-color: #ffffff;
+  }
+
+  .inline-code {
+    font-size: 15px;
+    font-weight: 600;
+    padding: 2px 5px;
+    border: 1px solid #eae9e9;
+    border-radius: 4px;
+    background-color: #f3f2f2;
+    color: #3a3a3a;
+  }
+</style>
+
+<main>
+  <h1>Welcome to storybook</h1>
+  <p>This is a UI component dev environment for your app.</p>

```

Comment:
Should we keep consistency on the content of the Welcome component? There is useful info in other components that I don't see here, such as telling users about hot module reloading, a link to storybook repo and explanation over the webpack config file.